cves/CVE-2003-0020.yml

```diff
@@ -201,4 +217,13 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This vulnerability was caused by code written decades ago. In those
+    years, software vulnerabilities were not as well known as they are today. It's
+    easy to imagine that the original author of the code was simply unaware of the
+    possibility of escape sequence injection through error logs. The most popular
+    mitigation for this type of vulnerability is input validation. The recommended
+    mitigation for CWE-150 is specifically whitelist input validation. However,
+    the author who wrote the fix for this vulnerability chose to implement code
+    more akin to a blacklist, rather than a whitelist, which was explicitly not
```

Comment:
upvote for the explicit statement of blacklisting input should not be the solution, even though this is how it was solved.